File: src/Python/Algorithms/math/prime/prime.py
def prime(n):
    sequence = []

    for i in range(2, n):
        isPrime = True
        
        for j in range(2, i):
            if i % j == 0:
                isPrime = False
                break

        if i > 1 & isPrime == True:
            sequence.append(i)

    return sequence

File: src/JavaScript/Algorithms/math/prime/prime.js
function prime(n) {
    const sequence = []

    for (let i = 2; i <= n; i++) {
        var isPrime = true;

        for (let j = 2; j < i; j++) {
            if (i % j == 0) {
                isPrime = false;
                break;
            }
        }

        if (i > 1 && isPrime == true) {
            sequence.push(i);
        }
    }
    return sequence
}